plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -143,6 +150,12 @@ class CliOptions {
             handler = ExplicitBooleanOptionHandler.class)
     private boolean useLatestAll = true;
 
+    @Option(name = "--archive", usage = "Set to true to check archived update center repositories for plugin dependencies metadata first. " +
+            "If the metadata is found in the archive, it will be used instead of the latest update center metadata. " +
+            "By default, the latest update center metadata is used.",
+            handler = ExplicitBooleanOptionHandler.class)
+    private boolean useArchiveAll = false;
```

Comment:
I think this is clearer
```suggestion
    private boolean useArchiveFirst = false;
```